tools/ci/templates/jobs-mrwebrtc-release-pack.yaml

```diff
@@ -29,9 +29,28 @@ jobs:
     - msbuild
   variables:
     projectRoot: 'external/webrtc-uwp-sdk/webrtc/windows/projects/msvc/'
-    packageName: 'Microsoft.MixedReality.WebRTC.Native.${{parameters.packagePlatform}}'
   steps:
 
+  # Compute package name
+  - task: PythonScript@0
+    displayName: 'Compute package name'
+    inputs:
+      scriptSource: inline
+      script: |
+        import sys;
+        platform = sys.argv[1];
+        print("Platform : %s" % platform);
+        if (platform == 'Desktop'):
+          packageName = "mrwebrtc"
```

Comment:
nit: should it be `mrwebrtc_win32` to be consistent with the project name?

Reply:
Looking at nuget.org, native non-UWP packages don't have any suffix. So trying to be consistent with that.